### PR TITLE
more accessible red buttons

### PR DESCRIPTION
**Pull request by jwunderl**
fix microsoft/pxt-microbit#2292; pulled out of https://github.com/microsoft/pxt/pull/5968 as the other targets didn't actually need to darken this button as much, just microbit

66072296-a360b080-e509-11e9-9b58-91cbda9ee44c

### CODE REVIEW

File: theme/site/elements/button.overrides [hunk 1]
```diff
@@ -1,3 +1,10 @@
 /*******************************
          Site Overrides
 *******************************/
+
+.ui.red.button {
```
Comment (pelikhan) [inline theme/site/elements/button.overrides]: should this fix go into pxt itself?

Comment (jwunderl) [reply theme/site/elements/button.overrides]: Issue is pretty specific to the red chosen for microbit (it's very pale); looking at it again there aren't that many places @red actually seems to be used, I'll go through and see if just changing that to make it slightly darker is a better idea

**Comment by jwunderl** [comment]
Hm, so only other explicit use was in the toolbox when deleting, which would be a bit darker:

before:


this:


Do we particularly care about that / should I add a rule to lighten it back up a bit, or does it seem fine?

**Comment by pelikhan** [comment]
still feel like this belong to pxt, not specific to microbit

**Comment by jwunderl** [comment]
The issue is specific to how light the chosen sui red variable is for microbit; normal sui red / etc is perfectly fine. the current one is closer to salmon / a light pink:

Screen Shot 2019-10-17 at 4 08 24 PM


arcade
Screen Shot 2019-10-17 at 3 59 47 PM

adafruit
Screen Shot 2019-10-17 at 4 00 15 PM

lego
Screen Shot 2019-10-17 at 4 00 55 PM

all pass / are fairly dark,

only other problematic one is minecraft which doesn't quite pass (about .5 off, vs about 2 off for microbit)
Screen Shot 2019-10-17 at 4 01 42 PM

**Pull request merged**